src/rovr_control/rovr_control/main_control_node.py

```diff
@@ -117,37 +122,32 @@ def __init__(self) -> None:
             self.autonomous_berm_location = create_pose_stamped(1.3, -0.6, 90)  # TODO: Test this location in simulation
             self.dig_location = create_pose_stamped(6.2, -1.2, 0)  # TODO: Test this location in simulation
 
-        # Define timers here
-        self.apriltag_timer = self.create_timer(0.1, self.start_calibration_callback)
-        self.apriltag_timer.cancel()  # Cancel the apriltag timer initially
-
         # Define service clients here
         self.cli_skimmer_toggle = self.create_client(SetPower, "skimmer/toggle")
         self.cli_skimmer_stop = self.create_client(Stop, "skimmer/stop")
         self.cli_skimmer_setPower = self.create_client(SetPower, "skimmer/setPower")
         self.cli_lift_setPosition = self.create_client(SetPosition, "lift/setPosition")
         self.cli_drivetrain_stop = self.create_client(Stop, "drivetrain/stop")
-        self.cli_drivetrain_drive = self.create_client(Drive, "drivetrain/drive")
+        self.cli_drivetrain = self.create_client(Drive, "drivetrain/drive")
         self.cli_drivetrain_calibrate = self.create_client(Stop, "drivetrain/calibrate")
         self.cli_motor_get = self.create_client(MotorCommandGet, "motor/get")
         self.cli_lift_stop = self.create_client(Stop, "lift/stop")
         self.cli_lift_zero = self.create_client(Stop, "lift/zero")
         self.cli_lift_set_power = self.create_client(SetPower, "lift/setPower")
-        self.cli_set_apriltag_odometry = self.create_client(ResetOdom, "resetOdom")
 
         # Define publishers and subscribers here
         self.drive_power_publisher = self.create_publisher(Twist, "cmd_vel", 10)
         self.joy_subscription = self.create_subscription(Joy, "joy", self.joystick_callback, 10)
         self.skimmer_goal_subscription = self.create_subscription(Bool, "/skimmer/goal_reached", self.skimmer_goal_callback, 10)
 
-        self.started_calibration = False
-        self.field_calibrated = False
+        self.field_calibrated: Future = Future()
+        self.field_calibrated.set_result(False)
         self.nav2 = BasicNavigator()  # Instantiate the BasicNavigator class
 
         # ----- !! BLOCKING WHILE LOOP !! ----- #
-        while not self.cli_lift_zero.wait_for_service(timeout_sec=1):
-            self.get_logger().warn("Waiting for the lift/zero service to be available (BLOCKING)")
-        self.cli_lift_zero.call_async(Stop.Request())  # Zero the lift by slowly raising it up
+        # while not self.cli_lift_zero.wait_for_service(timeout_sec=1):
+        #     self.get_logger().warn("Waiting for the lift/zero service to be available (BLOCKING)")
```

Comment:
Uncomment this before we merge this PR into main, we don't want this code commented out in main.

Reply:
fixed